src/Tizen.NUI/src/public/BaseComponents/TextFieldEvent.cs

```diff
@@ -193,10 +193,9 @@ public partial class TextField
         }
 
         /// <summary>
-        /// The SelectionChanged event.
+        /// The SelectionChanged signal is emitted whenever the selected text changed.
```

Comment:
/// The SelectionChanged signal is emitted whenever the selected text is changed.